pkg/sysclt/sysctl.go

```diff
@@ -0,0 +1,50 @@
+package sysctl
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// ValidateSysctls validates a list of sysctl and returns it.
+func ValidateSysctls(strSlice []string) (map[string]string, error) {
```

Comment:
Can we call it just `Validate`? Calling `sysctl.ValidateSysctls` seems redundant.

Reply:
SGTM